static/app/views/starfish/modules/databaseModule/queries.js

```diff
@@ -154,12 +154,12 @@ export const getPanelEventCount = (date_filters, row) => {
 };
 
 export const getMainTable = (
+  startTime,
```

Comment:
Why'd we rearrange this? 🤔

Reply:
I had a weird typing error, so I converted this file to `.ts`, fixed the typing error, my ide complained about it being out of order, so I reordered it, then converted it back to `.js` (to keep the PR more focused)

Reply:
lol ok

Reply:
yeah lol, maybe we just convert this to .ts later haha